README.md

```diff
@@ -13,22 +14,50 @@ docker run --rm \
   -v /tmp/.X11-unix:/tmp/.X11-unix \
   -v /dev/bus/usb:/dev/bus/usb \
   -v /usr/share/X11/xkb:/usr/share/X11/xkb:ro \
-  -v ~/.AndroidStudio:/home/developer/.AndroidStudio \
```

Comment:
Can you explain the reason behind these changes?

Reply:
Most of the changes are reflective of how the Android Studio binaries are located in my local installation, e.g. `-v ~/.AndroidStudio.sdk:/home/developer/Android/Sdk`, so you may overlook or rename accordingly. I found some of the mounts to be unnecessary, like `-v ~/.AndroidStudio:/home/developer/.AndroidStudio` and `-v ~/.AndroidStudio.android:/home/developer/.android`. The most important volume mount is `-v ~/.AndroidStudio.config:/home/developer/.config`, I noticed that it makes the configuration after first startup more persistent.